plasmapy/physics/distribution.py

```diff
@@ -143,6 +143,144 @@ def Maxwellian_1D(v,
         return distFunc
 
 
+def Maxwellian_velocity_2D(vx,
+                           vy,
+                           T,
+                           particle="e",
+                           Vx_drift=0,
+                           Vy_drift=0,
+                           vTh=np.nan,
+                           units="units"):
+    r"""
+    pdf of velocity for a 2D Maxwellian distribution.
```

Comment:
Re-phrasing as "pdf of velocity for a Maxwellian distribution in 2D" would make it clearer that this is specifically for things in two dimensions.

Reply:
done